docs/griptape-framework/data/vector-stores.md

```diff
@@ -152,4 +152,49 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
 8. `create_index(self, name: str)`: Creates a new index in the Marqo client with the given name.
 9. `delete_index(self, name: str)`: Deletes an index in the Marqo client with the given name.
 10. `get_indexes(self)`: Returns a list of all indexes in the Marqo client.
-11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
\ No newline at end of file
+11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
+
+## MongoDbAtlasVectorStoreDriver
```

Comment:
Did you mean to commit those changes as well?

Reply:
No changes were done to Marqodriver example. Only Mongodb was added.